packages/modules/itwin-viewer-react/package.json

```diff
@@ -42,12 +42,14 @@
   "dependencies": {
     "@bentley/icons-generic-webfont": "^1.0.15",
     "@bentley/itwin-error-handling-react": "3.0.0",
+    "@bentley/property-grid-react": "^1.2.1",
     "@microsoft/applicationinsights-react-js": "2.3.1",
     "@microsoft/applicationinsights-web": "2.3.1",
     "history": "^4.10.1",
     "node-sass": "^4.14.1",
     "oidc-client": "^1.10.1",
     "react-redux": "^7.2.0",
+    "react-spring": "^8.0.27",
```

Comment:
ill make a note to update the property-grid-react pkg to make this a direct dep there

Reply:
thanks!